docs/sg_execution_times.rst

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Should this file be here?

Reply:
Oh good catch. I'll add that to the `.gitignore`